code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -82,11 +82,7 @@
 	var/harvest_phrase_third = "%PERSON harvests... something... into %VESSEL."
 	// Dummy chemicals - called if chem_type is null.
 	var/list/dummy_chems = list(
-		/datum/reagent/abnormality/nutrition,
-		/datum/reagent/abnormality/cleanliness,
-		/datum/reagent/abnormality/consensus,
-		/datum/reagent/abnormality/amusement,
-		/datum/reagent/abnormality/violence,
+		/datum/reagent/abnormality/sin/emptiness,	//Set to a useless dummy chem rn. Should actually be replaced on every abno
```

Comment:
I hate to ask but why isnt the dummy chemical defaulting to one of the sin reagents. I know its easier to pin down abnormalities that are broken but it may be better to have mismatched abnormality chems so it doesnt give a useless debug chemical. Unless emptiness is useful for something.

Reply:
It's just to pinpoint broken abnos. I will give it a minor effect so it's not absolutely useless